mkosi/qemu.py

```diff
@@ -819,6 +819,9 @@ def finalize_kernel_command_line_extra(config: Config) -> list[str]:
         # CD-ROMs are read-only so tell systemd to boot in volatile mode.
         cmdline += ["systemd.volatile=yes"]
 
+    if config.qemu_gui and config.architecture.is_arm_variant():
+        cmdline += ["console=tty0"]
```

Comment:
This should be moved to `finalize_kernel_command_line_extra()`

Reply:
It's already in that function